accio-cache/src/main/java/io/accio/cache/CacheManager.java

```diff
@@ -355,28 +357,27 @@ private class Task
         private final TaskInfo taskInfo;
         private final CompletableFuture<?> completableFuture;
 
-        public Task(TaskInfo taskInfo, CompletableFuture<?> completableFuture)
```

Comment:
Because I want to change task status in `taskInfo` from `QUEUD` to `RUNNING` in `handleCache` function, it means I need to move `handleCache` function to Task constructor.